tests/src/JIT/Performance/CodeQuality/BenchmarksGame/k-nucleotide/k-nucleotide.cs

```diff
@@ -0,0 +1,284 @@
+﻿/* The Computer Language Benchmarks Game
+   http://benchmarksgame.alioth.debian.org/
+ *
+ * submitted by Josh Goldfoot
+ * 
+ */
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+[assembly: OptimizeForBenchmarks]
+[assembly: MeasureInstructionsRetired]
+[assembly: MeasureGCCounts]
+
+namespace BenchmarksGame
+{
+
+public class knucleotide
+{
+#if DEBUG
+    const int Iterations = 1;
+    const string InputFile = "knucleotide-input.txt"; 
+#else
+    const int Iterations = 10;
+    const string InputFile = "knucleotide-input-big.txt"; 
+#endif
+
+
+    static string FindInput(string s)
+   {
+       string CoreRoot = System.Environment.GetEnvironmentVariable("CORE_ROOT");
+
+       if (CoreRoot == null)
+       {
+           Console.WriteLine("This benchmark requries CORE_ROOT to be set");
+           return null;
+       }
+
+       string inputFile = s ?? InputFile;
+
+       // Normal testing -- input file will end up next to the assembly
+       // and CoreRoot points at the test overlay dir
+       string[] pathPartsNormal = new string[] {
+           CoreRoot, "..", "..", "JIT", "Performance",
+           "CodeQuality", "BenchmarksGame", "k-nucleotide", inputFile
```

Comment:
@guhuro actually, don't worry about fixing finding the input file for now - I talked with @DrewScoggins, and we can't do that until we're able to move perf tests away from netstandard1.4, which we can't now. I've pre-emptively disabled this test for Helix (https://github.com/dotnet/coreclr/pull/9315), and opened an issue to re-enable it when possible (https://github.com/dotnet/coreclr/issues/9314).